### PR TITLE
Add pointing component

### PR DESCRIPTION
**Pull request by veteran29**
**When merged this pull request will:**
- Allow to send camera direction to current vehicle crew via DUI Squad Radar

### CODE REVIEW

File: addons/pointing/XEH_postInit.sqf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#include "script_component.hpp"
+
+#include "initKeybinds.sqf"
+
+if (IS_MOD_LOADED(diwako_dui_radar)) then {
```
Comment (3Mydlo3) [inline addons/pointing/XEH_postInit.sqf:5]: Wouldn't it be better to include in common preInit just like you did with `ace_finger`?

File: addons/pointing/functions/fnc_keyPressVehicle.sqf [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#include "script_component.hpp"
+/*
+ * Author: veteran29
+ * Handle vehicle point keybind.
+ *
+ * Arguments:
+ * None
+ *
+ *
```
Comment (3Mydlo3) [inline addons/pointing/functions/fnc_keyPressVehicle.sqf:9]: Unnecessary empty line
```suggestion
```

File: addons/pointing/XEH_postInit.sqf (hunk 1 above)
Comment (veteran29) [reply addons/pointing/XEH_postInit.sqf:5]: I just assessed that it might not be so useful for other components as ACE stuff. Should I move it?

Comment (3Mydlo3) [reply addons/pointing/XEH_postInit.sqf:5]: You might be right. It can be done later if needed anyway.

**Pull request merged**